client/components/theme/index.jsx

```diff
@@ -197,14 +208,16 @@ export class Theme extends Component {
 		const themeImgSrcDoubleDpi = photon( screenshot, { fit, zoom: 2 } );
 		const e2eThemeName = name.toLowerCase().replace( /\s+/g, '-' );
 
+		const bookmarkRef = this.props.bookmarkRef ? { ref: this.props.bookmarkRef } : {};
+
 		return (
-			<Card className={ themeClass } data-e2e-theme={ e2eThemeName }>
+			<Card className={ themeClass } data-e2e-theme={ e2eThemeName } onClick={ this.setBookmark }>
 				{ this.isBeginnerTheme() && (
 					<Ribbon className="theme__ribbon" color="green">
 						{ translate( 'Beginner' ) }
 					</Ribbon>
 				) }
-				<div className="theme__content">
+				<div className="theme__content" { ...bookmarkRef }>
```

Comment:
I feel like in this case it could be more clear to do `ref={ bookmarkRef }` and then destructure `bookmarkRef` from props instead of putting it inside of an object. Though I'm not sure -- does `ref` need to be a value, or can it be null?

Reply:
This is what I did at first, and ref can be null.  What happens is that there is a `ref={null}` on every Theme card that way.  I am not sure if having the unnecessary `ref` prop on that many cards is "ok" (it probably is), but to be safe I edited it this way so that only the bookmarked theme card has a `ref` prop on it.  

React documentation talks about not 'overusing' refs, and while having `ref={null}` all over the place may not be what they are referring to, I figured limiting the prop to only the card that is actually using it would be the safest thing to do.

Reply:
sounds good!